code/modules/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -44,9 +49,14 @@
 				meat_counter++
 			H.gib()
 			obj_integrity = min(obj_integrity + max_integrity*0.05,max_integrity)//restores 5% hp of tendril
+			for(var/mob/living/L in view(src, 5))
+				if(L?.mind.has_antag_datum(/datum/antagonist/ashwalker))
```

Comment:
On second glance of this, L will never be null so you don't need the check at all
```suggestion
				if(L.mind.has_antag_datum(/datum/antagonist/ashwalker))
```